go/bundle/encoder.go

```diff
@@ -165,6 +165,36 @@ func (is *indexSection) Finalize(ver version.Version) error {
 		if err := enc.EncodeMap(mes); err != nil {
 			return err
 		}
+	} else {
+		// CDDL:
+		// index = {* whatwg-url => [ location-in-responses ] }
+		// whatwg-url = tstr
+		// location-in-responses = (offset: uint, length: uint)
+		m := make(map[string][]*indexEntry)
+		for _, e := range is.es {
+			url := e.URL.String()
+			m[url] = append(m[url], e)
+		}
+
+		mes := []*cbor.MapEntryEncoder{}
+		for url, es := range m {
```

Comment:
What happens if there are multiple indexEntries for a single URL?

Reply:
We will have a slice of indexEntries per url and (as far as I know) we just pick the first one when finding a matching response to a url.